gpio.py

```diff
@@ -102,25 +104,36 @@ def cleanup(pin=None, assert_exists=False):
 
 
 @_verify
-def setup(pin, mode, pullup=None, initial=False):
+def setup(pin, mode, pullup=None, initial=False, active_low=False):

```

Comment:
`None` and `None` needs to be a valid value.

Reply:
Not sure if this was a typo when you said `None` twice here. I changed the default value to be `None` and added logic that will preserve the original behavior if that is the value. This should preserve API and functional backwards compatibility.

Reply:
I have no idea what I meant, haha -- but what you did sounds like what I would mean if I had been meaning something remotely intelligent, as opposed to... what I said... :wink: